testsuite/MDAnalysisTests/converters/test_rdkit.py

```diff
@@ -396,7 +397,8 @@ def test_sanitize_fail_warning(self):
         u = mda.Universe(mol)
         with pytest.warns(UserWarning, match="Could not sanitize molecule"):
             u.atoms.convert_to.rdkit(NoImplicit=False)
-        with pytest.warns(None) as record:
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
```

Comment:
Is the idea that this block should pass without a warning but if it does then the test fails?

Well, the test fails, and I don't know why — this seems to be behavior that we masked previously.

Reply:
It's because we are expecting _some_ warnings just not that specific warning we're checking the record for. I've changed the simplefilter to a filterwarnings that should do the trick instead